src/components/App.jsx

```diff
@@ -5,15 +5,16 @@ import Header from './header/Header';
 import ManageLoginData from './ManageLoginData';
 import HomePage from './HomePage';
 import ManageRegisterData from './ManageRegisterData';
+import AuthLayout from './AuthLayout';
 
 const App = () => {
     return (
         <>
             <Header/>
             <Routes>
                 <Route path="/" element={<HomePage/>}/>
-                <Route path="/login" element={<ManageLoginData/>}/>
-                <Route path="/register" element={<ManageRegisterData/>}/>
+                <Route path="/login" element={<AuthLayout isRegistration={false} name="login"/>}/>
+                <Route path="/register" element={<AuthLayout isRegistration={true} name="register"/>}/>
```

Comment:
`isRegistration` seems superfluous. if `name="register"` is passed in, isn't this enough information to indicate to `AuthLayout` that it's a registration context?